h2o-py/h2o/explanation/_explain.py

```diff
@@ -1230,6 +1230,49 @@ def pd_multi_plot(
 
 def _center(col):
     col[:] = col - col[0]
+def _prepare_grouping_frames(frame, grouping_variable):
+    if (grouping_variable not in frame.names):
```

Comment:
nit/style: It's stylistically preferred to not wrap conditions with parenthesis except for long expressions. Ref e.g. https://google.github.io/styleguide/pyguide.html#33-parentheses

Reply:
fixed